modules/apps/portal-language/portal-language-override-service/service.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.4.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_4_0.dtd">
+
+<service-builder auto-import-default-references="false" auto-namespace-tables="false" mvcc-enabled="true" package-path="com.liferay.portal.language.override">
+	<namespace>PortalLanguageOverride</namespace>
+	<entity local-service="true" name="PLOEntry">
+
+		<!-- PK fields -->
+
+		<column name="ploEntryId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" />
```

Comment:
@drewbrokke why this needs to have the userName? It is duplicated to userId. And userName could change, leaving this info out of date.